.vscode/settings.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Please remove this file, it's generated by VSCode and not necessary for the repo